.github/workflows/automerge.yml

```diff
@@ -5,7 +5,7 @@ jobs:
   automerge:
     name: Automerge Dependabot
     runs-on: ubuntu-latest
-    if: github.actor == 'dependabot[bot]' || github.actor == 'dependabot-preview[bot]'
+    if: github.actor == 'dependabot[bot]' || github.actor == 'dependabot-preview[bot]' || github.event.label.name == 'dependencies'
```

Comment:
Are you sure the trick with the "dependencies" label works? According to the linked readme, this workflow is not executed at all unless a specific token is used.

Reply:
I created a token, added it to our secrets as `GH_TOKEN_UPDATE_GRADLE_WRAPPER`. This should be in line with the documentation.

Reply:
Ok, nice. But what usage is then the "dependencies" label? If you just want to have automerge for this PR, then I would say something like `github.event.pull_request.title` is better.